### DISCUSSION TITLE
[운영체제 3일차] 스레드

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#86] [오늘의 주제]
스레드 질문 2인조 1개, 3인조 2개

**Comment by zini9188** [comment]
스레드의 생명 주기에 대해 설명해주세요

**Comment by seongeun42** [comment]
프로세스와 스레드의 차이에 대해 설명해주세요.

**Comment by watty-exclamationMark** [comment]
스레드란 무엇이며,
User Level Threads (사용자 레벨 스레드)와 Kernel Level Threads(커널 레벨 스레드)에 대해 설명해주세요.

**Comment by kjc6735** [comment]
멀티 쓰레드의 동시성과 병렬성을 설명해주세요.

**Comment by seongeun42** [comment]
Thread-Safe라는 말은 무슨 의미인가요?
